### ISSUE TITLE
Adding the ability to retrieve the microphone input as a string

### ISSUE THREAD

**Issue by Gamemap** [godotengine/godot-proposals#2434]
### Describe the project you are working on
A game like Emergency 4
### Describe the problem or limitation you are having in your project
I want to receive the microphone signal as a sentence to run a function. 
(If you do not have enough time to select the objects with the mouse because too much is happening in the game at the same time).
### Describe the feature / enhancement and how it helps to overcome the problem or limitation
Get the microphone single as a string.
### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams
$AudioStreamMicrophone.get_as_string
### If this enhancement will not be used often, can it be worked around with a few lines of script?
I don't think so.
### Is there a reason why this should be core and not an add-on in the asset library?
It is used in many programms (also in some games). So I think it should be included in core or in an official add-on.

**Comment by Calinou** [comment]
It seems you're basically requesting speech-to-text support. Speech-to-text isn't a simple problem to solve (it's still far from perfect today), and I'd consider it to be add-on territory.

**Comment by Zireael07** [comment]
Yeah, looks like he wants speech to text. Definitely addon territory (and I say this as someone who often relies on STT)
